tests/interface/functors/Makefile.am

```diff
@@ -7,3 +7,4 @@
 check_LTLIBRARIES = libfunctors.la
 libfunctors_la_SOURCES = functors.cpp
 libfunctors_la_LDFLAGS = -avoid-version -shared -rpath /tmp
+libfunctors_la_CPPFLAGS = -I$(srcdir)/include
```

Comment:
`$(srcdir)` in this case is `tests/interface/functors`. If this PR compiles, and it seems to, then this flag is not needed.

Reply:
I was not sure - I will remove it. 
The point is that the functor library did not use Souffle includes before; with stateful functors we use `SymbolTable.h` and `RecordTable.h`.

Reply:
In that case you would need `$(top_srcdir)/src/include`. It compiled for me before because souffle had been installed on the machine I tried to compile on - so the headers were in the path.

`srcdir` is the directory the current Makefile.am is in
`top_srcdir` is the main souffle directory